src/plugins/prefabs/src/lib.rs

```diff
@@ -1,21 +1,29 @@
 mod systems;
+
+pub mod components;
 pub mod traits;
 
 use std::any::TypeId;
 
 use bevy::{
-	app::{App, Plugin},
+	app::{App, Plugin, PreUpdate},
 	asset::Handle,
 	pbr::StandardMaterial,
 	render::mesh::Mesh,
 };
 use common::resources::Shared;
+use systems::instantiate_children::instantiate_children;
 
 pub struct PrefabsPlugin;
 
+impl PrefabsPlugin {
+	const INSTANTIATION_FRAME: PreUpdate = PreUpdate;
```

Comment:
=> `INSTANTIATION_LABEL` ?

Reply:
Introduced a new `Labels` struct contain globals to synchronize systems across plugins.